tests/python/pytorch/graphbolt/impl/test_sampled_subgraph_impl.py

```diff
@@ -141,6 +147,123 @@ def test_exclude_edges_hetero(reverse_row, reverse_column):
     _assert_container_equal(result.original_edge_ids, expected_edge_ids)
 
 
+@pytest.mark.parametrize("reverse_row", [True, False])
+@pytest.mark.parametrize("reverse_column", [True, False])
+def test_exclude_edges_homo_csc_format(reverse_row, reverse_column):
```

Comment:
Since csc_format is the future default format, so I'd change this to test_exclude_edges_homo, and change existing test_exclude_edges_homo to test_exclude_edges_homo_node_pair.